__tests__/loader.test.js

```diff
@@ -67,6 +67,35 @@ describe('Loader', () => {
     });
   });
 
+  describe('#loadPattern', () => {
+    it('returns a pattern object', () => {
+      const patternPath = '${__dirname}/__fixtures__/patterns/pattern-valid.json';
+      return Loader.loadPattern(patternPath).then(pattern => {
+        expect(pattern).toBeDefined();
+      });
+    });
+
+    it('returns null if the file is invalid json', () => {
+      const patternPath = '${__dirname}/__fixtures__/patterns/pattern-valid.json';
```

Comment:
The test workflows aren't running on this PR so I haven't had a chance to verify tests locally yet but it seems that this one should load the invalid JSON file instead.

```suggestion
      const patternPath = '${__dirname}/__fixtures__/patterns/pattern-invalid.json';
```